src/main/java/salesforce/ui/pages/opportunities/OpportunitiesPopUpLightning.java

```diff
@@ -32,6 +38,17 @@ protected void waitUntilPageObjectIsLoaded() {
         webDriverWait.until(ExpectedConditions.elementToBeClickable(newButton));
     }
 
+    @Override
+    public void displayOpportunityList(final String listName) {
+        String opportunityOrderedListLocator;
+
```

Comment:
mayle you can declare this varible out of the method
and delete the 44 ompty line

Reply:
Why out, it was not have another call, it is not necessary.